modules/tracking/include/opencv2/tracking/tracker.hpp

```diff
@@ -1189,6 +1189,37 @@ class CV_EXPORTS_W TrackerTLD : public Tracker
   BOILERPLATE_CODE("TLD",TrackerTLD);
 };
 
+/** @brief KCF is a novel tracking framework that utilize properties of circulant matrix to enhance the processing speed.
+ * This tracking method is implementation of @cite KCF_ECCV which is extended to KFC with color-names features (@cite KCF_CN).
+ * The original paper of KCF is available at <http://home.isr.uc.pt/~henriques/circulant/index.html> 
+ * as well as the matlab implementation. For more information about KCF with color-names features, please refer to 
+ * <http://www.cvl.isy.liu.se/research/objrec/visualtracking/colvistrack/index.html>.
+ */
+class CV_EXPORTS_W TrackerKCF : public Tracker
+{
+ public:
+  enum MODE {GRAY, CN, CN2};
+  struct CV_EXPORTS Params
+  {
+    Params();
+    void read( const FileNode& /*fn*/ );
+    void write( FileStorage& /*fs*/ ) const;
+    
+    double sigma;			// gaussian kernel bandwidth
+    double lambda;			// regularization
+    double interp_factor;		// linear interpolation factor for adaptation
+    double output_sigma_factor;	// spatial bandwidth (proportional to target)    
+    bool resize;			// activate the resize feature to improve the processing speed
+    int max_patch_size;			// threshold for the ROI size
```

Comment:
I am not sure how to fix this, in my editor it's looks fine :yum: 
Let me try using another editor,...later